adobe_vipm/management/commands/process_3yc.py

```diff
@@ -0,0 +1,33 @@
+from django.core.management.base import BaseCommand
+from swo.mpt.extensions.core.utils import setup_client
+
+from adobe_vipm.flows.benefits import (
+    check_3yc_commitment_request,
+    resubmit_3yc_commitment_request,
+    submit_3yc_recommitment_request,
+)
+
+
+class Command(BaseCommand):
+    help = "Process 3-year commitment and recommitment requests"
+
+    def success(self, message):
+        self.stdout.write(self.style.SUCCESS(message), ending="\n")
+
+    def info(self, message):
+        self.stdout.write(message, ending="\n")
+
+    def handle(self, *args, **options):
```

Comment:
My friend, I think it is better here to have a date as an input parameter with default to `today()`. It always simplify maintenance if we compare with hardcoded `today()` in the code